components/favourites.js

```diff
@@ -93,121 +107,132 @@ export class Favourites extends Component {
     );
 
     return (
-      <Container id="favourites">
-        <Grid className={outerGrid} container spacing={24}>
-          <Grid item xs={6} className={buttons}>
-            <HeaderButton
-              id="backButton"
-              useLink
-              href={this.get_link("/benefits-directory")}
-              arrow="back"
-            >
-              {t("favourites.back_link")}
-            </HeaderButton>
-          </Grid>
-          <Grid item xs={6} className={cx(buttons, right)}>
-            <HeaderButton
-              href={this.props.printUrl}
-              target="print_page"
-              id="printButton"
-            >
-              <Print /> {t("Print")}
-            </HeaderButton>
-          </Grid>
+      <div className={outerDiv}>
+        <div className={whiteBanner}>
+          <Container className={topPadding}>
+            <div className={container2}>
+              <Grid container spacing={24}>
+                <Grid item xs={6}>
+                  <HeaderButton
+                    id="backButton"
+                    useLink
+                    href={this.get_link("/benefits-directory")}
+                    arrow="back"
+                  >
+                    {t("favourites.back_link")}
+                  </HeaderButton>
+                </Grid>
 
-          <Grid item xs={12} className={topMatter}>
-            <Header1 className={"BenefitsCounter"}>
-              {t("favourites.saved_benefits", { x: filteredBenefits.length })}
-            </Header1>
-          </Grid>
-          <Grid item md={8} xs={12}>
-            {this.state.showDisabledCookieBanner ? (
-              <DisabledCookiesBanner
-                t={t}
-                onClose={() =>
-                  this.setState({ showDisabledCookieBanner: false })
-                }
-              />
-            ) : null}
+                <Grid item xs={6} className={right}>
+                  <HeaderButton
+                    href={this.props.printUrl}
+                    target="print_page"
+                    id="printButton"
+                  >
+                    <Print /> {t("Print")}
+                  </HeaderButton>
+                </Grid>
+              </Grid>
+            </div>
+          </Container>
+        </div>
 
-            <Grid container spacing={24}>
```

Comment:
pretty much all the changes listed below were because I changed the indenting by adding a `<div>`